### PR TITLE
qm-cloudinit-dump: add page

### PR DESCRIPTION
**Pull request by abhisheksrivastava177**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/qm-cloudinitdump.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# qm cloudinit dump
```
Comment (navarroaxel) [inline pages/linux/qm-cloudinitdump.md:1]: The filename should be `qm-cloudinit-dump.md`

Comment (abhisheksrivastava177) [reply pages/linux/qm-cloudinitdump.md]: Done

File: pages/linux/qm-cloudinit-dump.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# qm cloudinit dump
+
+> Get automatically generated cloudinit config.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Get automatically generated cloudinit configs:
+
+`qm cloudinit dump {{vmid}} {{type}}`
```
Comment (navarroaxel) [inline pages/linux/qm-cloudinit-dump.md:8]: What is a `meta` config? :thinking: 
```suggestion
`qm cloudinit dump {{vm_id}} {{meta|network|user}}`
```

Comment (abhisheksrivastava177) [reply pages/linux/qm-cloudinit-dump.md]: I think it specifies that metadata should passed to cloud logs.

File: pages/linux/qm-cloudinit-dump.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# qm cloudinit dump
+
+> Get automatically generated cloudinit config.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Get automatically generated cloudinit configs:
```
Comment (navarroaxel) [inline pages/linux/qm-cloudinit-dump.md:6]: Perhaps this better
```suggestion
- Generate a cloudinit file for a specific configuration type:
```

File: pages/linux/qm-cloudinit-dump.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# qm cloudinit dump
+
+> Get automatically generated cloudinit config.
```
Comment (navarroaxel) [inline pages/linux/qm-cloudinit-dump.md:3]:
```suggestion
> Generate cloudinit configuration files.
```

Comment (abhisheksrivastava177) [reply pages/linux/qm-cloudinit-dump.md]: Done

File: pages/linux/qm-cloudinit-dump.md (hunk 3 above)
Comment (abhisheksrivastava177) [reply pages/linux/qm-cloudinit-dump.md]: Done

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**